src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Client/Pages/Authentication.razor

```diff
@@ -0,0 +1,39 @@
+@page "/authentication/{action}"
+@using Microsoft.AspNetCore.Components.WebAssembly.Authentication
+@using Microsoft.Extensions.Options
+
```

Comment:
Nit: preferable not to have a blank line here. We don't usually separate out the directives at the top into different blocks.